site/content/en/docs/contrib/tests.en.md

```diff
@@ -96,6 +96,10 @@ check functionality of minikube after evaluating podman-env
 #### validateStartWithProxy
 makes sure minikube start respects the HTTP_PROXY environment variable
 
+#### validateStartWithCustomCerts
+makes sure minikube start respects the HTTPS_PROXY environment variable
```

Comment:
and "custom certs" 
since this is a new test, consider adding details of how it runs

like 

a mitmproxy runs with a customer proxy alongside...this test runs only on ... then checks for ...

this could be example for other tests docs

Reply:
done. i've removed the entry in the docs so that the gen-docs github action can run as a test